docs/0.3/shared_spec.md

```diff
@@ -0,0 +1,7 @@
+# Shared Spec
+
+<!--
+  Copyright 2021 Cargill Incorporated
```

Comment:
This page probably should not be included (are we linking to it somewhere ?)

Reply:
updated